inc/utility/namespace.php

```diff
@@ -700,7 +700,7 @@ function format_bytes( $bytes, $precision = 2 ) {
  */
 function debug_error_log( $message, $message_type = null ) {
 	if ( defined( 'WP_TESTS_MULTISITE' ) === false && WP_DEBUG ) {
-		\trigger_error( $message, $message_type ); // @codingStandardsIgnoreLine
+		\trigger_error( $message, $message_type ?? E_USER_NOTICE ); // @codingStandardsIgnoreLine
```

Comment:
Debug error logs showed `PHP Warning:  Invalid error type specified in /srv/www/pressbooks.test/current/web/app/plugins/pressbooks/inc/utility/namespace.php on line 703` when trying to see export validation issues. Looks like we are not specifying any error type and `trigger_error` only accept integers as a error type, but not null.